flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/regression/MultipleLinearRegression.scala

```diff
@@ -107,6 +107,11 @@ class MultipleLinearRegression extends Predictor[MultipleLinearRegression] {
     this
   }
 
+  def setOptimizationMethod(optimizationMethod: String): this.type = {
```

Comment:
If you add options to an algorithm, then you have to adapt the code and online documentation.